ext/redhat/razor-server.env

```diff
@@ -7,4 +7,5 @@ JBOSS_CONFIG=standalone.xml
 JBOSS_MODULES_SYSTEM_PKGS=org.jboss.byteman
 LAUNCH_JBOSS_IN_BACKGROUND=true
 LANG=en_US.UTF-8
-JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
\ No newline at end of file
+JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
+HTTP_PORT=8080
```

Comment:
It's a small cost to add a newline add the end of files and it's [generally recommended to avoid weird issues with some tools](http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline).